pillow_retry/tasks.py

```diff
@@ -39,12 +40,20 @@ def process_pillow_retry(error_doc_id):
         except ValueError:
             # all fluff pillows have module path of 'fluff' so can't be imported directly
             _, pillow_class_name = pillow_class.rsplit('.', 1)
-            pillow = get_pillow_by_name(pillow_class_name)
+            try:
+                pillow = get_pillow_by_name(pillow_class_name)
+            except PillowNotFoundError:
+                pillow = None
 
         if not pillow:
-            notify_error("Could not find pillowtop class '%s' while attempting a retry." % pillow_class)
+            notify_error((
+                "Could not find pillowtop class '%s' while attempting a retry. "
+                "If this pillow was recently deleted then this will be automatically cleaned up eventually. "
+                "If not, then this should be looked into."
+            ) % pillow_class)
             try:
                 error_doc.total_attempts = PillowError.multi_attempts_cutoff() + 1
+                error_doc.save()
```

Comment:
@snopoke I think this was a bug that these wouldn't actually get saved. let me know if that's not right